core/src/main/java/io/questdb/griffin/engine/table/HeapRowCursor.java

```diff
@@ -35,11 +35,11 @@
  * from related cursor into queue until all cursors are exhausted.
  */
 class HeapRowCursor implements RowCursor {
-    private final IntLongPriorityQueue heap;
+    private final IntLongSortedList heap;
```

Comment:
nit: 
```suggestion
    private final IntLongSortedList list;
```

Reply:
Done in 6b10a5b48c92f63921e38e14535405523236f0ca